release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -0,0 +1,77 @@
+# CI Signal Lead Playbook
+
+TODO: This could be worded / structured better but I want it to live someplace more "real" than [a PR comment](https://github.com/kubernetes/test-infra/pull/5306#issuecomment-341277823)
+
+## Status Quo
+
+Currently, here's what I'm doing as the CI Signal role for release 1.9:
+
+### Checking [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) daily
+
+- if a job is failing in one of the meta-stages (Up, Down, DumpClusterLogs, etc), I find the owning sig
```

Comment:
Just want to stress to open issues for _everything_. If you talk to someone on slack and they say they're fixing it, still open an issue and label it with `/priority failing-test` for visibility. Not doing this bit me a couple times last release.